d2renderers/d2svg/d2svg.go

```diff
@@ -493,6 +493,10 @@ func renderOval(tl *geo.Point, width, height float64, style string) string {
 	return fmt.Sprintf(`<ellipse class="shape" cx="%f" cy="%f" rx="%f" ry="%f" style="%s" />`, cx, cy, rx, ry, style)
 }
 
+func renderDoubleOval(tl *geo.Point, width, height float64, style string) string {
+	return renderOval(tl, width, height, style) + renderOval(&geo.Point{X: tl.X + 5, Y: tl.Y + 5}, width-10, height-10, style)
```

Comment:
this should be a constant like https://github.com/terrastruct/d2/blob/bd15db4fdf77a6a1ef469489ffbee89089b83a17/d2renderers/d2svg/d2svg.go#L46